addons/website_event_sale/controllers/main.py

```diff
@@ -49,6 +49,11 @@ def _create_attendees_from_registration_post(self, event, registration_data):
                 event_ticket_id=ticket_id,
                 event_slot_id=slot_id,
             )
+            for product in ticket_sudo.linked_product_ids:
```

Comment:
what would happen hereabove if ticket is free but linked products are not?

Reply:
Ah... Well at this moment, it will ignore the product and just create the registration as we don't create an SO for free ticket.
For now I've added a constraint to check if we have additional products on free ticket. Will see if we keep it or not. I think that for the combo version, I was failing silently as we also needed an SO for them to work